arches/app/views/search.py

```diff
@@ -212,13 +215,13 @@ def export_results(request):
     app_name = settings.APP_NAME
 
     if (settings.DISABLE_EXPORT_FOR_ANONYMOUS_USER is True) and (request.user.username == "anonymous"):
-            message = _(
-                "Anonymous users are unable to download exports.  Please sign in with your {app_name} username and password."
-            ).format(**locals())
+        message = _("Anonymous users are unable to download exports.  Please sign in with your {app_name} username and password.").format(
```

Comment:
@RobOatesHistoricEngland - Just to check; have you formatted this with Black?  If it's above the line length limit then it'll get picked up by Stickler as an issue

Reply:
Black limits to 140 line length. I just let it do its thing.